v3/src/v2/codap-v2-types.ts

```diff
@@ -125,9 +125,145 @@ export interface ICodapV2TableStorage extends ICodapV2BaseComponentStorage {
+  enableMeasuresForSelection: boolean
+  isShowingCount: boolean
+  isShowingPercent: boolean
+  percentKind: number
+}
+
+interface ICodapV2ConectingLinesAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+}
+
+interface ICodapV2MovableValueAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  isShowingCount: boolean
+  isShowingPercent: boolean
+  valueModels: Record<string, number>[] // ??
+}
+interface ICodapV2MeanAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  equationCoordsArray: ICodapV2ProportionCoordinates[]
+}
+
+interface ICodapV2MedianAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  equationCoordsArray: ICodapV2ProportionCoordinates[]
+}
+
+interface ICodapV2StDevAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  equationCoordsArray: ICodapV2ProportionCoordinates[]
+}
+
+interface ICodapV2MadAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  equationCoordsArray: ICodapV2ProportionCoordinates[]
+}
+
+interface ICodapV2BoxPlotAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  equationCoordsArray: ICodapV2ProportionCoordinates[]
+  showOutliers: boolean
+  showICI: boolean
+}
+
+interface ICodapV2PlottedFunctionAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  adornmentKey: string
+  expression: string
+}
+
+interface ICodapV2PlottedValueAdornment {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  adornmentKey: string
+  expression: string
+}
+
+type ICodapV2Adornment = ICodapV2CountAdornment | ICodapV2ConectingLinesAdornment | ICodapV2MovableValueAdornment |
+                         ICodapV2MeanAdornment | ICodapV2MedianAdornment | ICodapV2StDevAdornment |
+                         ICodapV2MadAdornment | ICodapV2PlottedFunctionAdornment | ICodapV2PlottedValueAdornment |
+                         ICodapV2BoxPlotAdornment
+
+interface ICodapV2MovablePointStorage {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  coordinates: ICodapV2Coordinates
+}
+
+interface ICodapV2MovableLineStorage {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  isInterceptLocked: boolean
+  equationCoords: ICodapV2Coordinates | null
+  intercept: number
+  slope: number
+  isVertical: boolean
+  xIntercept: number
+}
+
+interface ICodapV2LSRL {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  isInterceptLocked: boolean
+  equationCoords: ICodapV2Coordinates | null,
+  showConfidenceBands: boolean
+}
+
+interface ICodapV2MultipleLSRLsStorage {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  showSumSquares: boolean
+  isInterceptLocked: boolean
+  showConfidenceBands: boolean
+  lsrls: ICodapV2LSRL[]
+}
+
+interface ICodapV2ValueModel {
+  isVisible: boolean
+  enableMeasuresForSelection: boolean
+  values: Record<string, number>
+}
+
+interface ICodapV2MultipleMovableValuesStorage {
```

Comment:
Do we need this and `ICodapV2MovableValueAdornment` above? They appear to be the same.

Reply:
Thanks for catching that. I've double checked and we do _not_ need both. I'm going to remove `ICodapV2MultipleMovableValuesStorage`, and do a little clean up of `ICodapV2MovableValueAdornment`.